XenAdmin/Wizards/PatchingWizard/AutomatedUpdatesBasePage.cs

```diff
@@ -572,7 +573,7 @@ private void buttonSkip_Click(object sender, EventArgs e)
                 if (!uploadedPatches.Contains(patch))
                 {
                     planActionsPerHost.Add(new DownloadPatchPlanAction(host.Connection, patch, AllDownloadedPatches, patchFromDisk));
-                    planActionsPerHost.Add(new UploadPatchToMasterPlanAction(host.Connection, patch, patchMappings, AllDownloadedPatches, patchFromDisk));
+                    planActionsPerHost.Add(new UploadPatchToMasterPlanAction(this, host.Connection, patch, patchMappings, AllDownloadedPatches, patchFromDisk));

```

Comment:
The `UploadPatchToMasterPlanAction` action should be called with `skipDiskSpaceCheck = true` here, because in the automated update mode we do the disk space check on the Prechecks page.